src/managers/auth-manager.js

```diff
@@ -70,6 +85,32 @@ var _findUserInGroup = function (members, id) {
 };
 
 AuthManager.prototype = $.extend(AuthManager.prototype, {
+
+    /**
+    * Logs user in.
+    *
+    * For [end users](../../../glossary/#users), the `group` is required in order to log in. You can pass it as part of the `options` parameter if it was not included when the Authorization Manager was instantiatied. 
+    *
+    * If some end users are in multiple groups, on the login page for your project you can call `getUserGroups()` (see details below), present the end users with a list of groups, and require that they select one.
+    *
+    * **Example**
+    *
+    *       authMgr.login({
+    *           account: 'acme-simulations', 
+    *           userName: 'enduser1', 
+    *           password: 'passw0rd' 
+    *       });
+    *
+    *       authMgr.login()
+    *           .then(function(authAdapter) {
+    *               authMgr.getUserGroups({userId: authAdapter.user.user_id});
+    *               // present user with list of groups
+    *           });
+    *
+    * **Parameters**
+    *
+    * @param {Object} `options` (Optional) Overrides for configuration options.
```

Comment:
Let's spell out options.account/project/group/.. here as well, and just say this will override options you pass in while instantiation. People are more likely going to ctrl+f to login to see what parameters it takes instead of reading the doc page top-down.